### PR TITLE
Start of New Freestyle Structure and Accept Button is changed to color success

### PR DESCRIPTION
**Pull request by flarinerin**
As a Developer I want an easier way to navigate documentation of controls.

### Acceptance Criteria
- [ ] It’s done when… freestyle side nav is better organized

### New PR Checklist

- [ ] Rebase before opening pr

### CODE REVIEW

**Comment by flarinerin** [comment]
Please review, I need this to complete my forms work in Denovo. @jweakley

**Comment by flarinerin** [comment]
Also updated submit button color to btn-success

**Comment by flarinerin** [comment]
Okay try it now, @jweakley

File: index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ module.exports = {
   name: 'ember-bootstrap-controls',
 
   isDevelopingAddon() {
-    return false;
+    return true;
```
Comment (jweakley) [inline index.js]:
```suggestion
    return false;
```

**Pull request merged**